$OpenBSD: patch-Makefile_bsd-wrapper,v 1.2 2014/09/17 09:04:38 sthen Exp $

Last hunk: patch to install CGIs as mode 700, so that pkg_create run as
non-root is able to add them to the package. They are still installed with
mode 000 by pkg_add because the PLIST has an "@mode 000" marker.

--- Makefile.bsd-wrapper.orig	Thu Mar 13 01:50:02 2014
+++ Makefile.bsd-wrapper	Thu Mar 13 01:52:23 2014
@@ -28,15 +28,15 @@ LNDIR=	/usr/bin/lndir
 #    proxycachedir: $prefix/proxy
 #</Layout>
 
-HTTPD_EXEC_PREFIX=	TRUEPREFIX
+HTTPD_EXEC_PREFIX=	${TRUEPREFIX}
 HTTPD_BINDIR=		${HTTPD_EXEC_PREFIX}/bin
 HTTPD_SBINDIR=		${HTTPD_EXEC_PREFIX}/sbin
 HTTPD_LIBEXECDIR=	${HTTPD_EXEC_PREFIX}/lib/apache/modules
 HTTPD_SYSCONFDIR=	${HTTPD_PREFIX}/conf
 HTTPD_ICONSDIR=		${HTTPD_PREFIX}/icons
 HTTPD_HTDOCSDIR=	${HTTPD_PREFIX}/htdocs
-HTTPD_MANUALDIR=	${HTTPD_EXEC_PREFIX}/share/apache-httpd-openbsd/doc/
-HTTPD_PREFIX=		${HTTPD_EXEC_PREFIX}/share/apache-httpd-openbsd/www
+HTTPD_MANUALDIR=	${HTTPD_EXEC_PREFIX}/share/doc/apache-httpd-openbsd
+HTTPD_PREFIX=		/var/www
 HTTPD_INCLUDEDIR=	${HTTPD_EXEC_PREFIX}/lib/apache/include
 HTTPD_RUNDIR=		${HTTPD_PREFIX}/logs
 HTTPD_LOGDIR=		${HTTPD_PREFIX}/logs
@@ -767,7 +767,7 @@ distribution:
 		j=`dirname $$i`; \
 		${INSTALL} -d -g ${BINGRP} -m 755 ${DESTDIR}${HTTPD_PREFIX}/$$j/; \
 		echo "Installing ${DESTDIR}${HTTPD_PREFIX}/$$i"; \
-		${INSTALL} ${INSTALL_COPY} -g ${BINGRP} -m 000 \
+		${INSTALL} ${INSTALL_COPY} -g ${BINGRP} -m 700 \
 		    ${.OBJDIR}/$$i ${DESTDIR}${HTTPD_PREFIX}/$$j/; \
 	done
 	@-for i in ${ICONFILES}; do \
